src/format/kdbx4/mod.rs

```diff
@@ -81,15 +81,12 @@ mod kdbx4_tests {
             password += &std::char::from_u32(random_char as u32).unwrap().to_string();
         }
 
-        let key_elements = DatabaseKey::new()
-            .with_password(&password)
-            .get_key_elements()
-            .unwrap();
+        let db_key = DatabaseKey::new().with_password(&password);
```

Comment:
A random question, but can this fail? What if empty password is entered?

Reply:
@varjolintu I think empty passwords are supported by the spec, and will result in different key elements than if no password is used for encryption.